src/bypasses/ouo.js

```diff
@@ -0,0 +1,24 @@
+import BypassDefinition from "./BypassDefinition.js"
+
+export default class Ouo extends BypassDefinition {
+    constructor() {
+        super()
+    }
+    execute() {
+        //Bypassing stuff goes here
+        if (location.pathname !== '/') {
+            if (/(go|fbc)/.test(location.pathname.split("/")[1])) {
+                document.querySelector("form").submit()
+            }
+            else {
+                if (document.querySelector("form#form-captcha")) {
+                    document.querySelector("form#form-captcha").action = `/xreallcygo${location.pathname}`
+                    document.querySelector("form#form-captcha").submit()
+                }
+                else {
+                    this.helpers.crowdBypass()
```

Comment:
```suggestion
                    //this.helpers.crowdBypass()
```

crowdBypass() has been deprecated, it will probably be replaced by 
```
this.helpers.crowdQuery(window.location.hostname, window.location.pathname.slice(1)).then((response) => {
    this.helpers.crowdNavigate(response.target)
});
```
Just comment out crowd crowdBypass for now, you can open a PR for adding crowdQuery once #955 is merged